docs/variables.md

```diff
@@ -184,6 +184,18 @@ Only available for `successCondition`
 | `response.body` | Response body (`string`) |
 | `response.headers` | Response headers (`map[string][]string`) |
 
+### CronWorkflow Templates
```

Comment:
```suggestion
### CronWorkflows 
```

This applies to all CronWorkflows.
Also "Templates" in this context refers to "template types", i.e. steps, DAGs, HTTP, etc